taos-optin/src/conn/mod.rs

```diff
@@ -98,7 +98,7 @@ impl RawTaos {
     pub fn query_a<'a, S: IntoCStr<'a>>(
         &self,
         sql: S,
-        fp: taos_async_query_cb,
```

Comment:
do not change C types/functions names.

Reply:
the modifcation about c function has been rolled back